Log before and after merge

fredkingham opened the pull request:
* log the old patient  the new patient before the merge. Log the combined patient after the merge.
* Remove some unused imports in merged_patient
* mock load_patient as otherwise we get errors in the logs when it tries to hit the upstream apis

Commit:
Removes an unused import statements in merge_patient

Commit:
Mocks load_patient in a unit test to stop excessive logging

Mock loader.load_patient in test_merge_patient so that we don't do
unnecessary noisy logging.

Commit:
Makes merge_patient log patients.to_dict

Makes merge_patient log the patients before and after the merge.

Commit:
Changes merge patient to log to a seperate file

Outside of standard logging, just while we want to easily make sure that
merge patient works as expected.

davidmiller commented on intrahospital_api/merge_patient.py:
```diff
@@ -22,6 +22,9 @@
 
 IGNORED_FIELDS = {"id", "episode", "patient", "previous_mrn"}
 
+# not *.log so it will not be picked up log rotate
```
have we double checked our configuration of logrotate?

Commit:
Changes merge_patient to log the old patient.to_json to seperate file

Changes it so that we log the old patient.to_json to a seperate file.

In the log we will log everything i.e. old patient json, new patient json, merge patient json

davidmiller commented:
Merge criteria:

1. merges.txt _literally_ just the json of the patient about to be deleted and a newline.
2. If you want to log the stuff that's currently being logged to the generic log files, go for it [as long as this doesn't include upstream labs - I don't think it does.]
